### PR TITLE
Front-End Sites Compatibility

### LINKED ISSUE #7643: Pantheon Front-End Sites Doc Update
Re: [Pantheon Front-End Sites](https://pantheon.io/docs//guides/decoupled-sites/faq/)

Priority: Low/Medium/High (choose one, remove the other options)

## Suggested Resolution

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes #7643 

## Summary

**[Front-end Sites FAQ](https://pantheon.io/docs/guides/decoupled-sites/faq/)** - Updated wording to use compatibility in place of support.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7657](https://pr-7657--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `45` | `49` | `4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7657` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41363/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41363/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7657--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7657--pantheon-docs.my.pantheonfrontend.website/docs/guides/decoupled-sites/faq/#frequently-asked-questions)

**Comment by wordsmither** [comment]
lgtm

File: source/content/guides/decoupled-sites/06-faq.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ reviewed: "2022-08-11"
...
 ### Can I use any CMS with Pantheon architecture for Front-End Sites?
 
-WordPress and Drupal are currently the only CMSs supported on Pantheon Decoupled. 
+WordPress and Drupal are currently the only CMSs compatible with Pantheon Decoupled. 
 
 
 ### What dependency versions are compatible with Front-End Sites?
 
-Pantheon Front-End Sites currently support:
+Pantheon Front-End Sites is currently compatible with:
```
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/decoupled-sites/06-faq.md:27]: @joa-pan I changed it from support to compatible. I thought Kit said that Pantheon doesn't support it but  is compatible with it. If you're saying it needs to go back to support, I'm not sure why the issue was created?

[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/decoupled-sites/06-faq.md:27]: @joa-pan I checked with Kit. You are correct. I'll change it back to the original.

**Pull request merged**